modules/jaeger-install-elasticsearch.adoc

```diff
@@ -30,7 +30,7 @@ Do not install Community versions of the Operators. Community Operators are not
 
 . Click *Install*.
 
-. On the *Create Operator Subscription* page, select *All namespaces on the cluster (default)*. This installs the Operator in the default `openshift-operators` project and makes the Operator available to all projects in the cluster.
+. On the *Create Operator Subscription* page, select *openshift-operators-redhat*.
```

Comment:
@JStickler Still I see all the existing contents are valid.
The only change I see is default namespace changed from `openshift-operators` to `openshift-operators-redhat`

![image](https://user-images.githubusercontent.com/1004403/83641744-8d8d1900-a5cb-11ea-802a-395e89db703b.png)